index.test-d.ts

```diff
@@ -28,7 +28,7 @@ import {
 } from '.';
 
 expectType<AllElectron | Remote>(api);
-expectType<boolean>(api.app.isPackaged);
+expectType<boolean | undefined>(api.app.isPackaged);
```

Comment:
Why is this `undefined`?

Reply:
In [electron-api.json](https://github.com/electron/electron/releases/download/v4.1.0/electron-api.json) `isPackaged` property isn't marked as `required`, so in the [electron/typescript-definitions](https://github.com/electron/typescript-definitions) generated electron.d.ts file it is added as:

```ts
interface App extends EventEmitter {
  (…)
  isPackaged?: boolean;
}
```

hence `boolean | undefined`.

Reply:
Ugh, the Electron and Node.js type definitions are just so bad...